cvmfs/sync_union.h

```diff
@@ -233,12 +252,12 @@ class SyncUnionOverlayfs : public SyncUnion {
                            const std::string &filename);
   void ProcessCharacterDevice(const std::string &parent_dir,
                               const std::string &filename);
-  std::string UnwindWhiteoutFilename(const std::string &filename) const;
+  std::string UnwindWhiteoutFilename(const SyncItem &entry) const;
   std::set<std::string> GetIgnoreFilenames() const;
-  virtual void ProcessFile(SyncItem *entry);
+  virtual void ProcessFile(SyncItem &entry);
```

Comment:
Ah! That is the reason, this was the only pointer parameter. Anyway, fixing this in the [pull request on top](https://github.com/cvmfs/cvmfs/pull/1150), to avoid a rebase-rage.